jobs/monitor_jobs.groovy

```diff
@@ -0,0 +1,153 @@
+import utilities.StatusUpdater
+
+dirs = [
+  [name: 'grafana', slackChannel: 'monitor'],
+  [name: 'influxdb', slackChannel: 'monitor'],
+  [name: 'telegraf', slackChannel: 'monitor'],
+]
+
+dirs.each { Map dir ->
+  dir.commitEnvVar = "${dir.name.toUpperCase().replaceAll('-', '_')}_SHA"
+}
+
+dirs.each { Map dir ->
+  [
+    // for each repo, create a <dir.name>-master and <dir.name>-pr job
+    [type: 'master', branch: 'master'],
+    [type: 'pr', branch: '${sha1}'],
+  ].each { Map config ->
+    isMaster = config.type == 'master'
+    isPR = config.type == 'pr'
+
+    name = isMaster ? dir.name : "${dir.name}-pr"
+    downstreamJobName = defaults.testJob[config.type]
```

Comment:
im not sure how we would even test this other than the components actually start.